rest-api/tools/import_organizations.py

```diff
@@ -156,6 +156,7 @@ def _entity_from_row(self, row):
     city = row.get(SITE_CITY_COLUMN)
     state = row.get(SITE_STATE_COLUMN)
     zip_code = row.get(SITE_ZIP_COLUMN)
+    latitude, longitude = self._get_lat_long_for_site(address_1, city, state)
```

Comment:
Remove the TODO on line 155.

Also, check for address_1 being none; in that case we shouldn't call this.